src/package/SlateEditor/SlateEditor.js

```diff
@@ -15,7 +15,11 @@ class SlateEditor extends Component {
   }
 
   onChange (state) {
-    this.setState({ state })
+    const { onChange } = this.props;
+    this.setState({ state });
+    if(onChange) {
```

Comment:
@abobwhite Just a preference and a convention that we have adopted on the project.. Nothing in particular with the JS..
https://medium.com/@kentcdodds/semicolons-in-javascript-a-preference-dd8fc8b80895